### PR TITLE
58 consistent private mode

### LINKED ISSUE #58: Consistent private mode
Estimate: 6HP

Feature: 
        As a user
        I want private mode labels to be consistent across all windows
        In order to reduce confusion

Scenario: 
        Given that multiple windows are opened 
        When I press “toggle private mode” in one window
        Then the label and state changes consistently across all windows

### LINKED ISSUE #59: TEST: Consistent private mode
See #58

### PR DESCRIPTION
**Pull request by martingraf4**
Closes #58 
Closes #59

- [x] High Test-Coverage (90%), GUI Code might be an exception
- [x] All Tests passing
- [ ] All files autoformatted (to be discussed)
- [ ] Codequality: not many long methods
- [ ] Codequality: Encapsulation (classes should be loosely coupled, law of demeter,...)
- [ ] Timetracking: Issue Timetracking is complete (Time spent)

### CODE REVIEW

**Review by WolfgangDaniel — DISMISSED** [review]
Sieht gut aus.

File: RichClipboard.package/RCStorage.class/class/newWithRichClipboard..st [hunk 1]
```diff
@@ -0,0 +1,9 @@
+as yet unclassified
```
Comment (zortax) [inline RichClipboard.package/RCStorage.class/class/newWithRichClipboard..st]: Kategorie fehlt (?)

File: RichClipboard.package/RCStorage.class/class/newWithRichClipboard..st [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+newWithRichClipboard: aRichClipboard
+	| storage |
+	
+	storage := RCStorage basicNew.
+	storage richClipboard: aRichClipboard.
+	storage initialize.
```
Comment (zortax) [inline RichClipboard.package/RCStorage.class/class/newWithRichClipboard..st]: Cascade? (muss in squeak gemacht werden wegen einrückungen)

File: RichClipboard.package/RCStorageTest.class/instance/testSendsNotificationOnAddedItem.st [hunk 3]
```diff
@@ -4,9 +4,9 @@ testSendsNotificationOnAddedItem
 	
 	storageItem :=  'Test1' .
 	receivedClipping := nil.
-	self storage
+	storage
 		when: #clippingsChanged 
 		send: #value
 		to: [receivedClipping := (storage getAll first) asString].
-	self storage add: storageItem.
+	storage add: storageItem.
```
Comment (zortax) [inline RichClipboard.package/RCStorageTest.class/instance/testSendsNotificationOnAddedItem.st]: Kaskade, falsche Einrückung

File: RichClipboard.package/RichClipboard.class/instance/destroyContextMenu.st [hunk 4]
```diff
@@ -0,0 +1,9 @@
+destroy
+destroyContextMenu
+	contextMenu ifNotNil: [ 
+		contextMenu delete.
+		contextMenu removeActionsForEvent: #requestPasteClipping.
```
Comment (zortax) [inline RichClipboard.package/RichClipboard.class/instance/destroyContextMenu.st]: Kaskade?

**Review by zortax — CHANGES_REQUESTED** [review]
Keinen einzigen unnötigen Punkt am Methodenende gefunden.
Impressive

**Review by zortax — APPROVED** [review]

**Review by WolfgangDaniel — APPROVED** [review]

**Pull request merged**